{"uuid":"19eee3a8-7084-4eed-b3af-57854c954a5d","historyId":"7c6c9bcc50929b47554684265adaec0e","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testRunner.DsalgoRunner"},{"name":"testClass","value":"testRunner.DsalgoRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testRunner.DsalgoRunner"},{"name":"host","value":"LAPTOP-2KQV9F30"},{"name":"thread","value":"9012@LAPTOP-2KQV9F30.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Expected alert to contain 'SyntaxError', but got: NameError: name 'search' is not defined on line 7","trace":"java.lang.AssertionError: Expected alert to contain 'SyntaxError', but got: NameError: name 'search' is not defined on line 7\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat stepDefinitions.ArraysStep.the_user_must_see_an_error_message_in_the_alert_window(ArraysStep.java:325)\r\n\tat âœ½.The user must see an error message in the alert window(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:69)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Arrays PracticeQuestion\""},{"name":"arg0","value":"\"Verify that user receives error for invalid python code on running \"Search the array\" question\""}],"start":1746229516125,"stop":1746229520563}